src/tests/App.test.js

```diff
@@ -8,6 +9,10 @@ const props = {
   text: {}
 };
 
+beforeAll(() => {
+  configure({ adapter: new Adapter() });
+});
+
```

Comment:
You could DRY up all the `Adapter` code by putting this in a file you point to with `setupTestFrameworkScriptFile` and removing from individual test files:

```js
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

beforeAll(() => {
  configure({ adapter: new Adapter() });
});
```

You’d then also have to figure out how to pull in `raf/polyfill` through that file.